templates/terraform/decoders/firestore_document.go.erb

```diff
@@ -0,0 +1,23 @@
+<%# The license inside this block applies to this file.
+	# Copyright 2020 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+// metadata is not present if the API returns an error
+if name, ok := res["name"]; ok {
+	re := regexp.MustCompile("^projects/[^/]+/databases/[^/]+/documents/(.+)$")
+	match := re.FindStringSubmatch(name.(string))
+  if len(match) > 0{
+		res["path"] = match[1]
```

Comment:
Can you comment somewhere inside the decoder that we use the decoder to add the path field?